web_client/Dockerfile

```diff
@@ -1,6 +1,7 @@
 # "dev" is the developpment image
 #FROM node:14 as dev
-FROM node:14-bullseye-slim as dev
+FROM timbru31/node-alpine-git:14 as dev
```

Comment:
I prefer sticking to official images, timbru31 is perhaps a nice person but she is less accountable if she add a backdoor to their image.

Reply:
done